<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>716</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inosis</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblCurrentMood">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Your current mood is:
</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lstMoods">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>101</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblTitle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Inosis</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>640</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMoods">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Moods</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lstPlaylist">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>191</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblPlaylist">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Playlist - Happy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddSong">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>580</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Song</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRemove">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>580</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Song</string>
    </property>
   </widget>
   <widget class="QComboBox" name="profiles">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddProfile">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>30</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add User</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtArtistName">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>480</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>insert artist name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSongName">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>510</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>insert song name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtUrl">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>540</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>insert youtube link</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblArtistName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Artist</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSongName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Song</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblUrl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>URL</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtAddProfile">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>30</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>   Enter Username</string>
    </property>
   </widget>
   <widget class="QWebEngineView" name="youtubeVideo" native="true">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>100</y>
      <width>631</width>
      <height>591</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>30</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Read Emotion</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>PyQt5.QtWebEngineWidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>exit_program()</slot>
  <slot>get_song_list()</slot>
  <slot>update_profile()</slot>
 </slots>
</ui>
